qiime/plugin/plugin.py

```diff
@@ -188,19 +188,19 @@ class PluginMethods(PluginActions):
     # TODO is `register` a better name now that functions are the only accepted
     # source (i.e. markdown support is gone)?
     def register_function(self, function, inputs, parameters, outputs, name,
-                          description, help_text=None):
+                          description, help_texts=None):
```

Comment:
Numpydoc is definitely an option. I think I'd rather delay this (keeping it as is, just a dict) and spend the effort thinking of a better registration API altogether.